pkg/workspaceapi/api.go

```diff
@@ -0,0 +1,178 @@
+package workspaceapi
+
+import (
+	"reflect"
+
+	"github.com/warptools/warpforge/wfapi"
+
+	"github.com/ipld/go-ipld-prime/datamodel"
+	"github.com/ipld/go-ipld-prime/node/bindnode"
+	"github.com/ipld/go-ipld-prime/schema"
+	"github.com/serum-errors/go-serum"
+)
+
+const (
+	ModuleInterestLevel_Query ModuleInterestLevel = "Query"
+	ModuleInterestLevel_Run   ModuleInterestLevel = "Run"
+)
+const (
```

Comment:
It might be questionable to put these new error codes here instead of in wfapi. 
TODO: Needs description comments.